values-schema.yaml

```diff
@@ -218,6 +218,40 @@ definitions:
         type: string
     title: Container image
     type: object
+  jobs:
+    description: |
+      You may pass an array of Job objects. The Kubernetes API accepts a Job specification, 
+      which is meant for run-to-completion apps (e.g. batch Jobs).
+    items:
+      description: |
+        You may pass a Job object. You must pass the following properties:
+
+          enabled: (type boolean)
+          env: (type array of strings, pattern-matched)
+          image: (type string, pattern-matched)
+          init: (type object)
+            image: (type string, pattern-matched)
+            script: (type string)
+      properties:
```

Comment:
Please add `secret` and `script` properties.

Reply:
`script` already exists right? `script: (type string)`

But that's good, I was just working off the `jobs.demo.yaml` @Morriz gave me.